mods/d2k/rules/defaults.yaml

```diff
@@ -304,7 +248,7 @@
 	Inherits: ^Plane
```

Comment:
Looks like this is as good a time as any to finally remove `^Helicopter` entirely.